cadcutils/cadcutils/net/auth.py

```diff
@@ -174,7 +258,7 @@ def _signal_handler(signal, frame):
             # if args.cert_filename is None:
             #    cert_filename = os.path.join(os.getenv("HOME", "/tmp"), ".ssl/cadcproxy.pem")
 
-            cert = get_cert(cert_server=args.cert_server,
+            cert = get_cert(cert_server=args.cert_server, user=args.user,
```

Comment:
No. I was trying to minimize the cases in which we prompt the user because this code is often called from batch scripts. So the only case in which user gets a prompt is when they use the username argument and no corresponding authentication can be found in a netrc file.